unicorn-play/src/test/scala/org/virtuslab/unicorn/StringPlayUnicorn.scala

```diff
@@ -1,8 +1,8 @@
 package org.virtuslab.unicorn
 
 import com.google.inject.{ Inject, Singleton }
-import slick.backend.DatabaseConfig
-import slick.driver.JdbcProfile
+import org.virtuslab.unicorn.SlickExports.JdbcProfile
```

Comment:
Correct - basically this imports change between 3.1 and 3.2 version branches - that's why I aliased them in `SlickExports.scala` (two versions - appropriate compiled per scala version)